src/containers/edit-profile/profile-tab/profile-tabs/profile-professional-tab/ProfileProfessionalTab.tsx

```diff
@@ -0,0 +1,163 @@
+import { useTranslation } from 'react-i18next'
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+import { styles } from './ProfileProfessionalTab.styles'
+import AppButton from '~/components/app-button/AppButton'
+import {
+  ButtonVariantEnum,
+  OpenProfessionalCategoryModalHandler,
+  ProfessionalCategoryWithActivationControls,
+  ProficiencyLevelEnum
+} from '~/types'
+import Box from '@mui/material/Box'
+import ProfessionalCategoryList from '~/containers/edit-profile/professional-category-list/ProfessionalCategoryList'
+import { useModalContext } from '~/context/modal-context'
+import AddProfessionalCategoryModal from '~/containers/edit-profile/add-professional-category-modal/AddProfessionalCategoryModal'
+import AddIcon from '@mui/icons-material/Add'
+import AboutTutorAccordion from '~/containers/edit-profile/about-tutor-accordion/AboutTutorAccordion'
+
+// TODO: replace mock data to real data
+const mockCategoriesData: ProfessionalCategoryWithActivationControls[] = [
+  {
```

Comment:
can you create only one item and then:
```
const categoriesMock = Array(3)
  .fill()
  .map((_, index) => ({
    ...categoryMock,
    ...change name and other fields
  }))
```

Reply:
I think we shouldn't create so complex solution here because mock data differs a lot and definitely will be removed when real data comes from te server